packages/qgrid-ngx-plugins/src/lib/data-provider/data-provider-strategy.ts

```diff
@@ -0,0 +1,13 @@
+import { GridModel } from '@qgrid/ngx';
```

Comment:
can u put all strategies directly to the data-provider folder? also remove please index.ts file, we don't use them

Reply:
done